utils/augmenting.py

```diff
@@ -112,4 +112,37 @@ def get_augmented_val_X_y(X, y, label):
 
 
 
-     
\ No newline at end of file
+def get_augmented_val_id():
```

Comment:
You are douplicating code from get_augmented_val_X_y. Best practise would be to use this function (get_augmented_val_id) in the X_y-function.